### PR TITLE
Add locate command

### PR DESCRIPTION
**Pull request by zhangriqi**
LocateCommand followed by INDEX, to show the location on Google Map

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -27,14 +27,15 @@
 public class MainWindow extends UiPart<Stage> {
 
     private static final String FXML = "MainWindow.fxml";
+    private static BrowserPanel browserPanel = new BrowserPanel();
```
Comment (jonleeyz) [inline src/main/java/seedu/address/ui/MainWindow.java:30]: Hey @zhangriqi, why do we need to make the browserPanel static here? Just wondering!

**Review by jonleeyz — COMMENTED** [review]
Looks great @zhangriqi! Passed all tests?

**Comment by jonleeyz** [comment]
@melvintzw, @Der-Erlkonig, comments?

**Comment by melvintzw** [comment]
Looks good! Are the google map results valid?

**Review by melvintzw — APPROVED** [review]
Great job! I've cloned and tried out the new feature on my machine, seems to work fine!

**Pull request merged**